package.json

```diff
@@ -69,6 +69,7 @@
     "focus-visible": "^5",
     "intl-messageformat": "^7",
     "lit-element": "^2",
+    "lodash-es": "^4.17.20",
```

Comment:
Typically we'll just specify `^4` unless there's something we absolutely need in minor/patch releases.